misc/acrn-config/board_config/misc_cfg_h.py

```diff
@@ -219,14 +219,29 @@ def generate_file(config):
         sos_bootarg_diff(sos_cmdlines, config)
         print("", file=config)
 
-    if board_cfg_lib.is_rdt_supported():
-        print("", file=config)
-        common_clos_max = board_cfg_lib.get_common_clos_max()
-        max_cache_clos_entries = common_clos_max
-        if board_cfg_lib.is_cdp_enabled():
-            max_cache_clos_entries = 2 * common_clos_max
+    common_clos_max = board_cfg_lib.get_common_clos_max()
+    max_mba_clos_entries = common_clos_max
+    max_cache_clos_entries = common_clos_max
+
+    if board_cfg_lib.is_cdp_enabled():
+        max_cache_clos_entries_cdp_enable = 2 * common_clos_max
+        print("#ifdef CONFIG_RDT_ENABLED", file=config)
+        print("#ifdef CONFIG_CDP_ENABLED", file=config)
+        print("#define MAX_CACHE_CLOS_NUM_ENTRIES\t{}U".format(max_cache_clos_entries_cdp_enable), file=config)
+        print("#else", file=config)
+        print("#define MAX_CACHE_CLOS_NUM_ENTRIES\t{}U".format(max_cache_clos_entries), file=config)
+        print("#endif", file=config)
+        print("#define MAX_MBA_CLOS_NUM_ENTRIES\t{}U".format(max_mba_clos_entries), file=config)
+        print("#endif", file=config)
+    else:
+        print("#ifdef CONFIG_RDT_ENABLED", file=config)
+        print("#define MAX_MBA_CLOS_NUM_ENTRIES\t{}U".format(max_mba_clos_entries), file=config)
+        print("#endif", file=config)
         print("#define MAX_CACHE_CLOS_NUM_ENTRIES\t{}U".format(max_cache_clos_entries), file=config)
```

Comment:
As MAX_MBA_CLOS_NUM_ENTRIES is part of CONFIG_RDT_ENABLED define so should MAX_CACHE_CLOS_NUM_ENTRIES be also part of CONFIG_RDT_ENABLED.

Reply:
Agree. Will do.